src/test/polymer/expression-scanner_test.ts

```diff
@@ -12,16 +12,17 @@
  * http://polymer.github.io/PATENTS.txt
  */
 
-
 import {assert} from 'chai';
+import * as estree from 'estree';
 
 import {HtmlParser} from '../../html/html-parser';
-import {scanDocumentForExpressions} from '../../polymer/expression-scanner';
+import {JavaScriptParser} from '../../javascript/javascript-parser';
+import {AttributeDatabindingExpression, parseExpressionInJsStringLiteral, scanDocumentForExpressions, TextNodeDatabindingExpression} from '../../polymer/expression-scanner';
 import {CodeUnderliner} from '../test-utils';
 
 suite('ExpressionScanner', () => {
 
-  suite('scan()', () => {
+  suite('scanning html for expressions', () => {
```

Comment:
Is there a test that demonstrates the expression is not scanned/returned unless its in a template?